### DISCUSSION TITLE
Incompatible settings in camera_settings.cfg

### DISCUSSION THREAD

**Discussion by MJ666** [Guilouz/Creality-Helper-Script-Wiki#433] [Issues]
The following setting will not work together and the the second one will result in an error as long the first is set to 1.

CAM_WHITE_BALANCE_TEMPERATURE_AUTO WHITE_BALANCE_TEMPERATURE_AUTO=1
CAM_WHITE_BALANCE_TEMPERATURE WHITE_BALANCE_TEMPERATURE=4600
 
The first one should be set to "0" or the second one should be removed from [delayed_gcode LOAD_CAM_SETTINGS] section.

**Comment by Guilouz** [comment]
Done.
